src/Altinn.Platform/Altinn.Platform.Register/Register/Altinn.Platform.Register.csproj

```diff
@@ -39,4 +33,6 @@
     <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\Altinn.Platform.Register.xml</DocumentationFile>
   </PropertyGroup>
 
+  <ProjectExtensions><VisualStudio><UserProperties appsettings_1json__JsonSchema="" /></VisualStudio></ProjectExtensions>
```

Comment:
This seems very VS spesific. Do we need it?

Reply:
@acn-sbuad  agree. removed